cmplibrary/src/main/java/com/sourcepoint/cmplibrary/ConsentLib.java

```diff
@@ -208,22 +220,93 @@ public void onConsentReady(String euConsent, String consentUUID) {
             }
 
             @Override
-            public void onMessageChoiceSelect(int choiceId, int choiceType) {
-                Log.d(TAG, "onMessageChoiceSelect: choiceId:" + choiceId + "choiceType: " + choiceType);
+            public void onAction(int choiceId, int choiceType) {
+                Log.d(TAG, "onAction: choiceId:" + choiceId + "choiceType: " + choiceType);
                 //noinspection SwitchStatementWithTooFewBranches
                 switch (choiceType) {
-                    case 12:
+                    case ActionTypes.SHOW_PM:
                         ConsentLib.this.choiceType = MESSAGE_OPTIONS.SHOW_PRIVACY_MANAGER;
+                        onShowPm();
+                        break;
+                    case ActionTypes.MSG_ACCEPT:
+                        onMsgAccepted();
+                        break;
+                    case ActionTypes.MSG_CLOSE:
```

Comment:
Is `MSG_CLOSE` the dismiss action? If so, wouldn't it be better to call it `MSG_DISMISS`?